File: src/org/genyris/core/URISymbol.java
package org.genyris.core;

import org.genyris.exception.GenyrisException;

public class URISymbol extends EscapedSymbol {

	public URISymbol(String newSym) {
		super(newSym);
	}
	public Symbol getBuiltinClassSymbol(Internable table) {
		return table.URISYMBOL();
	}
	public void acceptVisitor(Visitor guest) throws GenyrisException {
		guest.visitFullyQualifiedSymbol(this);
	}
}

File: src/org/genyris/core/URISymbol.kt
package org.genyris.core

import org.genyris.exception.GenyrisException

class URISymbol(newSym: String?) : EscapedSymbol(newSym) {
    override fun getBuiltinClassSymbol(table: Internable): Symbol? {
        return table.URISYMBOL()
    }

    @kotlin.Throws(GenyrisException::class)
    override fun acceptVisitor(guest: Visitor) {
        guest.visitFullyQualifiedSymbol(this)
    }
}
